src/ducks/ui.js

```diff
@@ -50,6 +52,10 @@ export const uiSlice = createSlice({
 			state.marketsAssetFilter = action.payload.marketsAssetFilter;
 			persistState('ui', { marketsAssetFilter: state.marketsAssetFilter });
 		},
+		setPairListDropdownIsOpen: (state, action) => {
+			state.pairListDropdownIsOpen = action.payload;
+			state.blurBackgroundIsVisible = action.payload;
```

Comment:
is there a reason why we need this in the reducer? isn't the dropdown a local state?

Reply:
Yeah I wasn't sure while I was architecting the component but now it's definitely useless.